.github/workflows/ssh-runner.yml

```diff
@@ -53,11 +53,21 @@ jobs:
       - name: NVIDIA-SMI
         run: |
           nvidia-smi
-      
+
+      - name: Store Slack infos
+        #because the SSH can be enabled dynamically if the workflow failed, so we need to store slack infos to be able to retrieve them during the waitforssh step
+        shell: bash
+        run: |
+          if [ "${{ secrets[format('{0}_{1}', github.actor, 'SLACK_ID')] }}" != "" ]; then
+            echo "SLACKCHANNEL=${{ secrets[format('{0}_{1}', github.actor, 'SLACK_ID')] }}" >> $GITHUB_ENV
+          else
+            echo "SLACKCHANNEL=${{ secrets.SLACK_CIFEEDBACK_CHANNEL }}" >> $GITHUB_ENV
```

Comment:
As we're adding this to the GITHUB_ENV, is this less secure? Previously the information was stored in secrets - will making it persistent like this make it more accessible?

Reply:
No, it would still be masked by GitHub Actions.

At least, it follows what has been done by Guillaume.

Reply:
we need to test. normally github keep it obfuscated on the logs.
if it's not the case, eventually we can add this kind of command : `echo "::add-mask::$the_secret"
`

Reply:
let's try this PR

Reply:
FYI: there is a run page

https://github.com/huggingface/transformers/actions/runs/10736292447/job/29775922690

In its raw log page, searching `SLACKCHANNEL`, all places are masked `***`